CondCore/CondDB/plugins/RelationalAuthenticationService.cc

```diff
@@ -16,15 +16,17 @@
 #include <memory>
 #include <sys/stat.h>
 
-#include <boost/bind.hpp>
+#include <boost/bind/bind.hpp>
```

Comment:
The `std::` has a bind and a `std::function` as well.

Reply:
@Dr15Jones  Do you have further comment on this PR?

Reply:
@Dr15Jones is correct. We did a campaign a year ago but it didn't cover this one. I'm just fixing a warning in this PR, but would support a migration by @l2_db